style.css

```diff
@@ -450,3 +474,261 @@ textarea:invalid:required {
   margin: 0 auto;
   width: 4rem;
 }
+
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+  }
+
```

Comment:
- [ ] Please adjust your styling, see this image below compared to the original design.
- [ ]  Your paragraph text should be center, kindly fix that
- [ ]  the paragraph font-weight does not much the design (yours is bold), consider using the same as provided in the template, please
- [ ] the width of your input fields are not well adjusted, please fix them up
- [ ]  Please add a placeholder to your `input name and email field` as you did with the `textarea`, kindly remove the value you just set to them.
<img width="810" alt="center" src="https://user-images.githubusercontent.com/79058364/159717145-23657653-8420-47ba-a2ab-cd5486304370.png">

- [ ] It should be like this ⏬  instead 
<img width="791" alt="foorm" src="https://user-images.githubusercontent.com/79058364/159717331-143772f1-952a-4376-81dd-fd18b7fbedc0.png">

- [ ] Please give some breathing to this section, kindly  move these arrows a bit to the left as shown below ; following the figma template, you should consider using same padding ang margin.
<img width="690" alt="arr" src="https://user-images.githubusercontent.com/79058364/159721814-0d3554c6-e668-448c-bada-2e414741e4c1.png">